app/views/partials/team_members_details_table_filter_form.scala.html

```diff
@@ -58,20 +64,37 @@
 
 @views.html.helper.form(action = formAction) {
 
-    @search_and_filter(form, withFilter =false, searchLabelKey = "group.members.list.filter.label")
+    @search_and_filter(
+        form,
+        withFilter = false,
+        searchLabelKey = "group.members.list.filter.label"
+    )
 
     @if(teamMembers.nonEmpty) {
 
-        @if(withSearchPrefix(str = "", formFilter = None, formSearch = form("search").value) != "") {
-                @h2(withSearchPrefix(str = "", formFilter = None, formSearch = form("search").value))
+        @paginationMetaData.fold {
+            @p(msgs("common.total.team-members", teamMembers.length), id = Some("team-members-in-group"))
+        }{ paginationMeta =>
+            @defining((paginationMeta.currentPageNumber - 1) * paginationMeta.pageSize) { pageStart =>
+                <p class="govuk-body">
+                    @span(
+                        msgs("paginated.team-members.showing",
+                            pageStart + 1,
+                            pageStart + paginationMeta.currentPageSize,
+                            paginationMeta.totalSize
+                        )
+                    )
+                    @if(form.data.nonEmpty && form.data.get("search").isDefined) {
+                        @span(msgs("paginated.team-members.for.search", form.data.get("search").get))
+                    }
+                </p>
+            }
         }
 
-        <div class="govuk-!-margin-bottom-8 scrollable-table-container" id="members" tabindex="0">
-
         @table(
             caption = Some("common.teamMembers"),
             captionClasses = "govuk-visually-hidden",
-            id = Some("sortable-table"),
+            id = Some("team-members-table"),
```

Comment:
change this to "members" for the skip link, since we've removed the scrollable container (matches the not found heading below)

Reply:
done